File: files/CH-043.R
library(tidyverse)
library(readxl)

input = read_excel("files/CH-043 Sort Table columns .xlsx", range = "B2:G9")
test  = read_excel("files/CH-043 Sort Table columns .xlsx", range = "J2:O9")

result =   input %>% select(Regions, names(sort(colSums(select(., -Regions)), decreasing = TRUE)))

identical(result, test)
# [1] TRUE

File: files/000-099/043/CH-043.py
import pandas as pd
import re

input = pd.read_excel('CH-043 Sort Table columns .xlsx', usecols="B:G", skiprows=1, nrows = 7)
test = pd.read_excel('CH-043 Sort Table columns .xlsx', usecols="J:O", skiprows=1, nrows = 7)
test.columns = test.columns.str.replace(r'\.1$', '', regex=True)
                     
result = input[['Regions'] + input.drop('Regions', axis=1).sum().sort_values(ascending=False).index.tolist()]

print(result.equals(test)) # True